### DISCUSSION TITLE
API as a sole interactor with a model

### DISCUSSION THREAD

**Discussion by rkb965** [posit-conf-2023/vetiver#37] [Q&A] [answered]
This was all so cool, and it was mostly very new to me. If you're the only person interacting with a model, does it still make sense to add the API layer (rather than living in model object land) or is that overkill since APIs are about... sharing?

**Comment by juliasilge** [comment, marked as answer]
That's an interesting question! I think if I was the only person to interact with a model, I might still use pins as a way to track versions and be able to use the model in a convenient way even after I am done with model training, but I don't think I would deploy an API.

**Comment by rkb965** [comment]
Great, thanks for clarifying! I don't want to unnecessarily complicate things... but I'm happy to add a little more effort to a workflow if it's for a good cause. I wasn't sure where sole-model-interaction might land so thank you for elaborating!
